julia.html.markdown

```diff
@@ -478,67 +478,67 @@ catch e
 end
 
 # You can define functions that take keyword arguments
-function keyword_args(;k1=4, name2="hello")  # note the ;
+function keywordArgs(;k1=4, name2="hello")  # note the ;
     return Dict("k1" => k1, "name2" => name2)
 end
-# => keyword_args (generic function with 1 method)
+# => keywordArgs (generic function with 1 method)
 
-keyword_args(name2="ness")  # => ["name2"=>"ness", "k1"=>4]
-keyword_args(k1="mine")     # => ["name2"=>"hello", "k1"=>"mine"]
-keyword_args()              # => ["name2"=>"hello", "k1"=>4]
+keywordArgs(name2="ness")  # => ["name2"=>"ness", "k1"=>4]
+keywordArgs(k1="mine")     # => ["name2"=>"hello", "k1"=>"mine"]
+keywordArgs()              # => ["name2"=>"hello", "k1"=>4]
```

Comment:
Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194